pages/docs/reference/delegated-properties.md

```diff
@@ -153,7 +153,45 @@ fun main() {
 If you want to intercept assignments and "veto" them, use [`vetoable()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.properties/-delegates/vetoable.html) instead of `observable()`.
 The handler passed to the `vetoable` is called _before_ the assignment of a new property value has been performed.
 
-## Storing Properties in a Map
+## Delegating to another property
+
+A property can delegate its getter and setter to another property of the same class. In this case, use the property name
+with the `this::` qualifier as the delegate name.
```

Comment:
Not only `::this` and not only the same class. See all possible options in this test: https://github.com/JetBrains/kotlin/blob/ea1e16e309429ec63c8e619ec5db2c6fe1485e0b/libraries/stdlib/test/properties/delegation/PropertyReferenceTest.kt#L27